### PR TITLE
Added Comment Functionality

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: bin [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 9087aee42b23ac8ae80f4fd87124e419f957a47e
```
Comment (amshali) [inline bin]: what is this file? 
I don't think we need this. Please remove it and recommit.

File: app/controllers/Application.java [hunk 2]
```diff
@@ -178,9 +178,9 @@ public static void deleteComment(Long id, Long userId) {
     news(userId);
   }
 
-  public static void postComment(Long commentableId, String author, String content) {
```
Comment (amshali) [inline app/controllers/Application.java:183]: What is this?

File: .project [hunk 3]
```diff
@@ -15,11 +15,12 @@
 		<nature>org.eclipse.jdt.core.javanature</nature>
 		<nature>org.playframework.playclipse.playNature</nature>
 	</natures>
-	<linkedResources><link><name>secure</name><type>2</type><location>/home/amshali/docs/academic/CS378/play-1.2.4/modules/secure/app</location></link>
-<link><name>conf/secure</name><type>2</type><location>/home/amshali/docs/academic/CS378/play-1.2.4/modules/secure/conf</location></link>
-<link><name>public/secure</name><type>2</type><location>/home/amshali/docs/academic/CS378/play-1.2.4/modules/secure/public</location></link>
-<link><name>crud</name><type>2</type><location>/home/amshali/src/play-1.2.4/modules/crud/app</location></link>
-<link><name>conf/crud</name><type>2</type><location>/home/amshali/src/play-1.2.4/modules/crud/conf</location></link>
-<link><name>public/crud</name><type>2</type><location>/home/amshali/src/play-1.2.4/modules/crud/public</location></link>
+	<linkedResources><link><name>crud</name><type>2</type><location>C:/play-1.2.4/modules/crud/app</location></link>
```
Comment (jtessler) [inline .project:18]: Didn't we add .project to .gitignore? Why is this showing up here?

File: app/controllers/Application.java [hunk 4]
```diff
@@ -178,9 +178,9 @@ public static void deleteComment(Long id, Long userId) {
...
-    Commentable parent = Commentable.findById(commentableId);
-    parent.addComment(author, content);
+  public static void postComment(Long commentableId, Long userId, String content) {
```
Comment (amshali) [inline app/controllers/Application.java]: This method must go into a different controller class for handling comments. 
Question is why do we need to pass the userId? We can get it by getting the current logged in user information.

File: app/controllers/Application.java (hunk 2 above)
Comment (ghost) [inline app/controllers/Application.java:183]: This renders the page with the new comment.

File: bin (hunk 1 above)
Comment (ghost) [inline bin]: i'm not too sure. i didn't notice it there when i committed. i'll remove it.

File: app/controllers/Application.java (hunk 4 above)
Comment (amshali) [inline app/controllers/Application.java]: Please refactor this functionality into a new controller, let's call it Comments. and fix the routes and resubmit.

Comment (ghost) [inline app/controllers/Application.java]: i wanted to render the page of the user being viewed and not the current logged in user. This way when you comment it keeps you at the same page that you were viewing. The method was already in Application.java. We can easily create a new controller for this tho.

File: .project (hunk 3 above)
Comment (ghost) [inline .project:18]: I checked my .gitignore but .project is in there. Not sure why it's showing.

**Pull request merged**